libs/deso-protocol/src/lib/identity/deso-workspace-fork.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Did you mean to push this file? Seems like its specific to your local workspace

Reply:
nah i deleted it